CMakeLists.txt

```diff
@@ -157,8 +219,6 @@ set_common_compile_options(${PROJECT_NAME})
 set_target_properties(${PROJECT_NAME} PROPERTIES
     VERSION
         ${PROJECT_VERSION}
-    SOVERSION
-        ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}
```

Comment:
Please don't remove it, the soversion is important.

Reply:
Ok, just a rebasing mistake.